proto/umee/leverage/v1/tx.proto

```diff
@@ -37,6 +37,7 @@ service Msg {
   rpc Borrow(MsgBorrow) returns (MsgBorrowResponse);
 
   // MaxBorrow allows a user to borrow the maximum amount of tokens their collateral will allow.
+  // Zero is returned if there is no more tokens to borrow.
```

Comment:
This is a transaction, not a query. The return is error (not zero) on fail. `ErrMaxBorrowZero`.

Reply:
Imagine cosmwasm wants to do it. I don't think the transaction should fail. There is nothing wrong, we can handle it without failing transaction.

Reply:
In general, if transaction doesn't lead to incorrect state, it shouldn't fail. So in fact this is a fix.